contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol

```diff
@@ -200,7 +183,7 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
 
     uint256 juelsPerGas = _getJuelsPerGas(gasPriceWithOverestimation);
     uint256 estimatedGasReimbursement = juelsPerGas * executionGas;
-    uint256 fees = uint256(donFee) + uint256(adminFee);
+    uint72 fees = donFee + adminFee;
 
```

Comment:
In theory, this could overflow if both are set to 72.max. since you'll be using them as 96 anyway, why not be safe and cast them to 96 before using